### DISCUSSION TITLE
Have you experiences with large amount of documents

### DISCUSSION THREAD

**Discussion by Br4thering** [LeoKlaus/Paperparrot#139] [Q&A] [answered]
I have a large collection of documents ( abt. 3.5 GB) 10.000 + files
Do you have any experience on application performance for such a large library on no ngx server  MacOS ?

**Comment by LeoKlaus** [comment, marked as answer]
I don't. 
As Core Data internally uses SQLite, performance with search operations could become slower. In my experience, 3000+ documents work well, and there's no technical reason as to why it wouldn't work (albeit a bit slower). 10.000 records isn't that much in database terms.

The overview of documents is loaded lazily, so there's no performance impact there.
